src/core/parseHTML.js

```diff
@@ -15,7 +15,7 @@ define([
 // defaults to document
 // keepScripts (optional): If true, will include scripts passed in the html string
 jQuery.parseHTML = function( data, context, keepScripts ) {
-	if ( !data || typeof data !== "string" ) {
+	if ( typeof data !== "string" ) {
 		return null;
 	}
```

Comment:
The docs say the first arg is a String and that it returns Array. I don't think we should ever be returning null, despite the unit test. :smile:  If the first arg is not a string it seems either all bets are off or we should turn it into a string for cases we feel are possible (numbers, `null`, `undefined`)? I would be fine with removing this entirely but have a feeling it would be like our `jQuery.parseJSON(null)` fiasco.